### ISSUE TITLE
[Issue]: Hardware accelerated decoding - Enabling VP9 breaks HEVC?

### ISSUE THREAD

**Issue by therealmelee** [jellyfin/jellyfin#7274]
### Please describe your bug

Playing an HEVC 10 bit 4k mkv file via Chrome browser and Jellyfin web.

With VP9 **disabled** under Hardware **Decoding**, I have QSV working and get ~2x speed.

If I **enable** VP9 and restart the exact same file, hardware transcoding does not take place, encoding speed is <1x and CPU utilization maxes out.

Working / not working confirmed via CPU utilization and also intel_gpu_top.

The file itself is not VP9 and has nothing to do with VP9 that I can tell. The ffmpeg logs look identical except for the actual encoding speed, and both show as using h264_qsv as the encoder.

Also, regardless, per vainfo, my CPU should be able to perform hardware decoding of VP9:

```
vainfo: VA-API version: 1.12 (libva 2.10.0)
vainfo: Driver version: Intel iHD driver for Intel(R) Gen Graphics - 21.1.1 ()
vainfo: Supported profile and entrypoints
      VAProfileNone                   : VAEntrypointVideoProc
      VAProfileNone                   : VAEntrypointStats
      VAProfileMPEG2Simple            : VAEntrypointVLD
      VAProfileMPEG2Simple            : VAEntrypointEncSlice
      VAProfileMPEG2Main              : VAEntrypointVLD
      VAProfileMPEG2Main              : VAEntrypointEncSlice
      VAProfileH264Main               : VAEntrypointVLD
      VAProfileH264Main               : VAEntrypointEncSlice
      VAProfileH264Main               : VAEntrypointFEI
      VAProfileH264Main               : VAEntrypointEncSliceLP
      VAProfileH264High               : VAEntrypointVLD
      VAProfileH264High               : VAEntrypointEncSlice
      VAProfileH264High               : VAEntrypointFEI
      VAProfileH264High               : VAEntrypointEncSliceLP
      VAProfileVC1Simple              : VAEntrypointVLD
      VAProfileVC1Main                : VAEntrypointVLD
      VAProfileVC1Advanced            : VAEntrypointVLD
      VAProfileJPEGBaseline           : VAEntrypointVLD
      VAProfileJPEGBaseline           : VAEntrypointEncPicture
      VAProfileH264ConstrainedBaseline: VAEntrypointVLD
      VAProfileH264ConstrainedBaseline: VAEntrypointEncSlice
      VAProfileH264ConstrainedBaseline: VAEntrypointFEI
      VAProfileH264ConstrainedBaseline: VAEntrypointEncSliceLP
      VAProfileVP8Version0_3          : VAEntrypointVLD
      VAProfileVP8Version0_3          : VAEntrypointEncSlice
      VAProfileHEVCMain               : VAEntrypointVLD
      VAProfileHEVCMain               : VAEntrypointEncSlice
      VAProfileHEVCMain               : VAEntrypointFEI
      VAProfileHEVCMain10             : VAEntrypointVLD
      VAProfileHEVCMain10             : VAEntrypointEncSlice
      VAProfileVP9Profile0            : VAEntrypointVLD
      VAProfileVP9Profile2            : VAEntrypointVLD
```

I can disable/enable this setting and performance clearly follows this change.

### Jellyfin Version

Other

### if other:

10.8.0 alpha 5

### Environment

```markdown
- OS: Ubuntu 21.10
- Virtualization: Proxmox
- Clients: Chrome
- Browser: Chrome
- FFmpeg Version: 4.4.1 Jellyfin
- Playback Method: Web browser
- Hardware Acceleration: QSV
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking:
- Storage:
```

### FFmpeg logs

```shell
Extracts from ffmpeg log for working and broken hardware transcoding.

[VP9 ENABLED - NOT WORKING]
Stream mapping:
  Stream #0:0 (hevc) -> setparams (graph 0)
  Stream #0:4 (pgssub) -> scale (graph 0)
  overlay (graph 0) -> Stream #0:0 (h264_qsv)
  Stream #0:1 -> #0:1 (truehd (native) -> aac (native))
Press [q] to stop, [?] for help
[matroska,webm @ 0x55fba9dba740] sub2video: using 3840x2076 canvas
Output #0, hls, to '/mnt/data1/Proxmox/transcoding/78e143f4e922fa0f93bb7817ae5503ce.m3u8':
  Metadata:
    encoder         : Lavf58.76.100
  Stream #0:0: Video: h264, nv12(tv, bt709, progressive), 3840x2076 [SAR 1:1 DAR 320:173], q=2-31, 28150 kb/s, 23.98 fps, 90k tbn (default)
    Metadata:
      encoder         : Lavc58.134.100 h264_qsv
    Side data:
      cpb: bitrate max/min/avg: 28150425/0/28150425 buffer size: 56300850 vbv_delay: N/A
  Stream #0:1: Audio: aac (LC), 48000 Hz, 5.1, fltp (24 bit), 640 kb/s (default)
    Metadata:
      encoder         : Lavc58.134.100 aac
frame=    2 fps=0.0 q=0.0 size=N/A time=00:00:00.00 bitrate=N/A speed=   0x    
frame=   14 fps=0.0 q=26.0 size=N/A time=00:00:00.38 bitrate=N/A speed=0.626x    
frame=   26 fps= 23 q=26.0 size=N/A time=00:00:00.87 bitrate=N/A speed=0.763x    
frame=   38 fps= 23 q=26.0 size=N/A time=00:00:01.53 bitrate=N/A speed=0.925x    
frame=   51 fps= 24 q=26.0 size=N/A time=00:00:02.00 bitrate=N/A speed=0.927x    
frame=   63 fps= 23 q=26.0 size=N/A time=00:00:02.43 bitrate=N/A speed=0.904x    
frame=   76 fps= 24 q=26.0 size=N/A time=00:00:02.96 bitrate=N/A speed=0.927x    
[hls @ 0x55fba9e89bc0] Opening '/mnt/data1/Proxmox/transcoding/78e143f4e922fa0f93bb7817ae5503ce0.ts' for writing
frame=   90 fps= 24 q=25.0 size=N/A time=00:00:03.54 bitrate=N/A speed=0.955x    
frame=  102 fps= 24 q=24.0 size=N/A time=00:00:04.05 bitrate=N/A speed=0.963x    
frame=  115 fps= 24 q=26.0 size=N/A time=00:00:04.58 bitrate=N/A speed=0.971x    
frame=  128 fps= 24 q=29.0 size=N/A time=00:00:05.16 bitrate=N/A speed=0.988x    
frame=  141 fps= 25 q=21.0 size=N/A time=00:00:05.67 bitrate=N/A speed=0.99x    
[hls @ 0x55fba9e89bc0] Opening '/mnt/data1/Proxmox/transcoding/78e143f4e922fa0f93bb7817ae5503ce1.ts' for writing
frame=  154 fps= 25 q=25.0 size=N/A time=00:00:06.20 bitrate=N/A speed=0.994x    
frame=  167 fps= 25 q=27.0 size=N/A time=00:00:06.76 bitrate=N/A speed=   1x    
frame=  180 fps= 25 q=25.0 size=N/A time=00:00:07.29 bitrate=N/A speed=   1x    
frame=  193 fps= 25 q=21.0 size=N/A time=00:00:07.85 bitrate=N/A speed=1.01x    
frame=  206 fps= 25 q=24.0 size=N/A time=00:00:08.38 bitrate=N/A speed=1.01x    
frame=  218 fps= 25 q=18.0 size=N/A time=00:00:08.91 bitrate=N/A speed=1.01x    

[WORKING]
Stream mapping:
  Stream #0:0 (hevc) -> setparams (graph 0)
  Stream #0:4 (pgssub) -> scale (graph 0)
  overlay_qsv (graph 0) -> Stream #0:0 (h264_qsv)
  Stream #0:1 -> #0:1 (truehd (native) -> aac (native))
Press [q] to stop, [?] for help
[matroska,webm @ 0x55ec827c7880] sub2video: using 3840x2076 canvas
Output #0, hls, to '/mnt/data1/Proxmox/transcoding/7fb430a1e60232635c4c0db15c8355bf.m3u8':
  Metadata:
    encoder         : Lavf58.76.100
  Stream #0:0: Video: h264, qsv(tv, bt709, progressive), 3840x2076 [SAR 1:1 DAR 320:173], q=2-31, 28150 kb/s, 23.98 fps, 90k tbn (default)
    Metadata:
      encoder         : Lavc58.134.100 h264_qsv
    Side data:
      cpb: bitrate max/min/avg: 28150425/0/28150425 buffer size: 56300850 vbv_delay: N/A
  Stream #0:1: Audio: aac (LC), 48000 Hz, 5.1, fltp (24 bit), 640 kb/s (default)
    Metadata:
      encoder         : Lavc58.134.100 aac
frame=    2 fps=0.0 q=0.0 size=N/A time=00:00:00.00 bitrate=N/A speed=   0x    
frame=   22 fps=0.0 q=26.0 size=N/A time=00:00:00.74 bitrate=N/A speed=1.25x    
frame=   42 fps= 38 q=26.0 size=N/A time=00:00:01.53 bitrate=N/A speed=1.39x    
frame=   63 fps= 39 q=26.0 size=N/A time=00:00:02.43 bitrate=N/A speed=1.51x    
[hls @ 0x55ec82826080] Opening '/mnt/data1/Proxmox/transcoding/7fb430a1e60232635c4c0db15c8355bf0.ts' for writing
frame=   90 fps= 43 q=24.0 size=N/A time=00:00:03.56 bitrate=N/A speed=1.69x    
frame=  118 fps= 45 q=29.0 size=N/A time=00:00:04.71 bitrate=N/A speed=1.81x    
frame=  145 fps= 47 q=25.0 size=N/A time=00:00:05.84 bitrate=N/A speed=1.88x    
[hls @ 0x55ec82826080] Opening '/mnt/data1/Proxmox/transcoding/7fb430a1e60232635c4c0db15c8355bf1.ts' for writing
frame=  171 fps= 47 q=27.0 size=N/A time=00:00:06.93 bitrate=N/A speed=1.92x    
frame=  199 fps= 48 q=25.0 size=N/A time=00:00:08.08 bitrate=N/A speed=1.96x    
[hls @ 0x55ec82826080] Opening '/mnt/data1/Proxmox/transcoding/7fb430a1e60232635c4c0db15c8355bf2.ts' for writing
frame=  225 fps= 49 q=9.0 size=N/A time=00:00:09.21 bitrate=N/A speed=1.99x    
frame=  250 fps= 49 q=13.0 size=N/A time=00:00:10.21 bitrate=N/A speed=1.99x    
frame=  277 fps= 49 q=17.0 size=N/A time=00:00:11.39 bitrate=N/A speed=2.02x    
etc.
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by nyanmisaka** [comment]
It has been fixed by #7138 in nightly build.

**Comment by therealmelee** [comment]
Awesome, good to know that it wasn't me, and that it's already fixed! Thanks.
